src/OrganizationsSDKAdapter.js

```diff
@@ -0,0 +1,70 @@
+import {ReplaySubject, defer} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {deconstructHydraId} from '@webex/common';
+import {OrganizationsAdapter} from '@webex/component-adapter-interfaces';
+
+// TODO: Figure out how to import JS Doc definitions and remove duplication.
+/**
+ * A Member object that is part of a membership
+ *
+ * @external Organization
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/OrganizationsAdapter.js#L6}
+ */
+
+/**
+ * The `OrganizationsSDKAdapter` is an implementation of the `OrganizationsAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a organization.
+ *
+ * @implements {OrganizationsAdapter}
+ */
+export default class OrganizationsSDKAdapter extends OrganizationsAdapter {
+  constructor(datasource) {
+    super(datasource);
+
+    this.organizationObservables = {};
+  }
+
+  /**
+   * Fetches the organization data and returns in the shape required by adapter.
+   *
+   * @param {string} orgID ID of the organization for which to fetch data
+   * @returns {Promise<Organization>} Information about the organization of the given ID
+   *
+   * @private
+   */
+  fetchOrganization(orgID) {
+    return this.datasource.request({
+      service: 'hydra',
+      resource: `organizations/${orgID}`,
+    }).then((res) => res.body);
+  }
+
+  /**
+   * Returns an observable that emits organization data of the given ID.
+   *
+   * @param {string} ID ID of organization to get
+   * @returns {external:Observable.<Organization>} Observable stream that emits organization data
+   */
+  getOrg(ID) {
+    if (!(ID in this.organizationObservables)) {
+      // use ReplaySubject cause we don't need to set an initial value
+      this.organizationObservables[ID] = new ReplaySubject(1);
+
+      defer(() => this.fetchOrganization(ID)).pipe(
+        map((response) => ({
+          ID: deconstructHydraId(response.id).id,
```

Comment:
Why are you deconstructing the ID? You should send back the "Hydra ID"

Reply:
Right, the "hydra IDs" are the public API IDs, which are an encoded version of the backend UUIDs + metadata. When interacting with some internal services we need to deconstruct it but not when exposing it outwards.